demo/pages/modal/acceptBid.tsx

```diff
@@ -152,6 +152,7 @@ const AcceptBidPage: NextPage = () => {
       </div>
 
       <AcceptBidModal
+      currency={'0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359'}
```

Comment:
we shouldn't deploy this, it will cause issues on the deployed demo

Reply:
yeah